.ci.yaml

```diff
@@ -523,15 +523,6 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Linux fuchsia_precache
```

Comment:
So we can delete the code in the engine builders that uploads this stuff?

What about the code that builds it?

Reply:
Not sure about engine builders, probably no since google3 artifacts still roll through flutter.

Reply:
the tooling itself isn't used though